cmd/ethrex/cli.rs

```diff
@@ -141,6 +141,12 @@ pub fn cli() -> Command {
                 .value_parser(clap::value_parser!(EVM))
                 .help("Has to be `levm` or `revm`"),
         )
+        .arg(
```

Comment:
we should have a `import` subcommand instead of a `--only_import` flag. Not only it makes more sense but we're also closer to geth who also has this: https://geth.ethereum.org/docs/fundamentals/command-line-options

Reply:
Created https://github.com/lambdaclass/ethrex/issues/2122